packages/blade/src/components/Radio/__tests__/Radio.native.test.tsx

```diff
@@ -60,4 +61,29 @@ describe('<Radio />', () => {
     fireEvent.press(radio);
     expect(radio.props.accessibilityState.checked).toBeTruthy();
   });
+
+  it('should expose native element methods via ref', () => {
+    let refValue = null;
+    const Example = (): React.ReactElement => {
+      const ref = React.useRef<HTMLInputElement>(null);
+      return (
+        <RadioGroup label="Hello">
+          <Radio
+            value="apple"
+            ref={(value) => {
+              console.log(value);
+              // @ts-expect-error
+              ref.current = value;
+              refValue = value;
```

Comment:
why this?

Reply:
https://github.com/razorpay/blade/pull/949#discussion_r1071866113